src/utils/mod.rs

```diff
@@ -1,9 +1,741 @@
 //!  Utility functions for Rustup
 pub(crate) mod notifications;
+pub(crate) mod notify;
```

Comment:
I think we should put the module declarations below the imports.

Reply:
@djc Can I do that in a separate commit? I was trying to minimize the LoDiff back then...

Reply:
Sure.